.github/workflows/binary_artifact.yml

```diff
@@ -1,71 +1,67 @@
 name: Build binary artifacts
 
 on:
+  pull_request:
+    paths:
+      - '**/binary_artifact.yml'
   push:
     tags:
       - weekly.**
       - 0.**
 
-jobs:
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{ github.ref_name != 'master' }}
 
+jobs:
   build-linux:
     runs-on: ubuntu-20.04
     env:
       CC: gcc
       ZIPNAME: v_linux.zip
     steps:
       - uses: actions/checkout@v4
-      - name: Compile
+      - name: Compile release binaries
         run: |
           make
           ./v -skip-unused -cc $CC -prod -o v cmd/v
           ./v -skip-unused -cc $CC -prod cmd/tools/vup.v
           ./v -skip-unused -cc $CC -prod cmd/tools/vdoctor.v
       - name: Remove excluded
-        run: |
-           rm -rf .git/
-           rm -rf thirdparty/tcc/.git/
-           rm -rf vc/
-           rm -rf v_old
-           rm -rf vlib/v/tests/bench/gcboehm/*.svg
+        run: rm -rf .git/ thirdparty/tcc/.git vc/ v_old vlib/v/tests/bench/gcboehm/*.svg
```

Comment:
Restore the previous script. It is deliberately written that way, so that new entries can be added cleanly, the resulting diff is smaller, and the resulting line is not too long.